lib/connectors/kv-memory.js

```diff
@@ -118,6 +118,24 @@ function(modelName, key, ttl, options, callback) {
   process.nextTick(callback);
 };
 
+KeyValueMemoryConnector.prototype.ttl =
+function(modelName, key, options, callback) {
+  var store = this._getStoreForModel(modelName);
+
+  // key is unknown
+  if (!(key in store)) {
```

Comment:
You should check if the item is expired and remove it, the same way as `get` works.

Please extract L63-L69 into a shared method that can be used by both `get` and `ttl`.